src/node/waku_node.py

```diff
@@ -112,6 +177,10 @@ def ensure_ready(self):
         self.info_response = self.info()
         logger.info("REST service is ready !!")
 
+    @retry(stop=stop_after_delay(2), wait=wait_fixed(0.1), reraise=True)
```

Comment:
if we are not raising any error here then the @retry decorator has no purpose

Reply:
Registration still could exit with non zero code. Temporary connection problem to Infura or another problem.

Reply:
yes but the way this decorator works is that you decorate a function that may fail for some time until a certain condition is met
but this function will never fail, os.path.exists returns True or False

so in order for the decorator to make sense you would have something like 

```
@retry(stop=stop_after_delay(2), wait=wait_fixed(0.1), reraise=True)
def rln_credential_store_ready(self, creds_file_path):
        if not os.path.exists(creds_file_path):
              raise FileNotFound
```
in this case the decorator will repeat the function(each 0.1 s) if the file is not found for a max of 2 seconds
if after 2 seconds the file is still not found it will return an error

Reply:
Thanks for your suggestion. Working well after https://github.com/waku-org/waku-interop-tests/pull/16/commits/5c168fd06be1f6a21d1d93c0465bbb20c7a66d9c